FirmataParser.cpp

```diff
@@ -87,33 +98,35 @@ void FirmataParser::parse(uint8_t inputData)
       switch (executeMultiByteCommand) {
         case ANALOG_MESSAGE:
           if (currentAnalogCallback) {
-            (*currentAnalogCallback)(multiByteChannel,
+            (*currentAnalogCallback)(this,
```

Comment:
`this` should be `currentAnalogCallbackContext`

same for all other `this` context parameters